wasm/jit/engine.go

```diff
@@ -34,20 +35,43 @@ type engine struct {
 	compiledWasmFunctions     []*compiledWasmFunction
 	compiledWasmFunctionIndex map[*wasm.FunctionInstance]int64
 	// Store the host functions and indexes.
-	hostFunctions     []hostFunction
-	hostFunctionIndex map[*wasm.FunctionInstance]int64
+	compiledHostFunctions     []*compiledHostFunction
+	compiledHostFunctionIndex map[*wasm.FunctionInstance]int64
 }
 
-type hostFunction = func(ctx *wasm.HostFunctionCallContext)
-
 func (e *engine) Call(f *wasm.FunctionInstance, args ...uint64) (returns []uint64, err error) {
+	prevFrame := e.callFrameStack
+	defer func() {
+		if v := recover(); v != nil {
+			top := e.callFrameStack
+			var traces []string
+			var counter int
+			for top != prevFrame {
+				traces = append(traces, fmt.Sprintf("\t%d: %s", counter, top.getFunctionName()))
+				top = top.caller
+				counter++
+				// TODO: include DWARF symbols.
```

Comment:
Add reference to #58 

```suggestion
				// TODO: include DWARF symbols. See #58 
```